crates/bin/pcli/src/command/migrate.rs

```diff
@@ -30,6 +51,23 @@ pub enum MigrateCmd {
     /// minus any gas prices for the migration transaction.
     #[clap(name = "balance")]
     Balance,
+    /// Migrate balances from specified subaccounts to a destination wallet.
+    ///
+    /// All assets from the specified source subaccounts will be sent to the destination wallet.
+    /// A FullViewingKey must be provided for the destination wallet.
+    /// Gas fees will be paid from the source subaccount with the most fee token.
+    #[clap(name = "subaccount-balance")]
+    SubaccountBalance {
+        /// Range of source subaccount indices to migrate from (e.g., 0..17)
+        #[clap(long, required = true, value_parser = parse_range)]
+        from: std::ops::Range<u32>,
```

Comment:
Ideally I'd like to see support for arbitrary indices, rather than a range, which can be specified as optional repeating args with a min length of 1. That way, one could use `pcli migrate subaccount 4 62` and only the subaccounts 4 and 62 would be migrated, with nothing in between.

This is a nice-to-have, though, so let's wait to implement until we have a need for it. For the time being, range support is a marked improvement and a huge timesaver, especially for the threshold signing workflow.